hazelcast-hibernate/hazelcast-hibernate4/src/main/resources/META-INF/services/com.hazelcast.NodeInitializer

```diff
@@ -0,0 +1 @@
+com.hazelcast.instance.ExtendedNodeInitializer
```

Comment:
Sorry, that's a mistake. It shouldn't be included in this pull request.

We use custom `NodeExtension`, in the new parlance, to remove the `WanReplicationService`, which otherwise starts 16 threads and never uses them. Since WAN replication is an enterprise feature, it would be nice if there was a simple way to turn it off. Or, even better, if it was just off automatically when you don't have a license for it. Since it's not, we override `createService` to return a no-op `WanReplicationService` implementation.